stdlib/public/runtime/ErrorObject.mm

```diff
@@ -42,14 +42,28 @@
 using namespace swift;
 using namespace swift::hashable_support;
 
+// Mimic the memory layout of NSError so things don't go haywire when we
+// switch superclasses to the real thing.
```

Comment:
Can we add an assert that this does in fact match at runtime? It's probably enough to just assert that the instance sizes are the same.

Reply:
There is such an assert! Which is how I knew to do this :D

Reply:
Oh, good. 😃